packages/react/src/Button/Button.tsx

```diff
@@ -8,7 +8,7 @@ import styles from './Button.module.css'
 
 export type ButtonProps<C extends React.ElementType> = BaseProps<C> & {
   as?: C
-  variant?: 'primary' | 'secondary'
+  variant?: 'primary' | 'secondary' | 'subtle'
```

Comment:
Can you add a new Storybook story for this too please? We'll need that to generate a visual snapshot too.

Reply:
I responded in slack but it seems that the update snapshots command is broken. Maybe during the migration the file needed was moved. I will look into it but let me know if you figure out the issue here.